demisto_sdk/commands/generate_docs/README.md

```diff
@@ -37,7 +37,12 @@ Skip generating of breaking changes section.
 * If no `output` given, the README.md file will be generated in the `input` file repository.
 * If no `additionalinfo` is provided for a commonly-used parameter (for example, `API Key`), a matching default value
   will be used, see the parameters and defaults in `default_additional_information.json`.
-
+* If an **incident mirroring** is implemented in an integration make sure that the following configuration params (if in use) are named as stated in order for the **incident mirroring** section to be generated:
+  * Incidents fetch query
+  * Mirroring tag
+  * Mirroring Direction
+  * Close Mirrored XSOAR Incident
+  * Close Mirrored *Integration Display Name* Incident or Detection
```

Comment:
```suggestion
  * Close Mirrored *Integration Display Name* Incident
```
Detection it specific to CrowdStrike Falcon mirroring.
Determining this field should be rethought as it is called differently to match different incident types. In CrowdStrike Falcon it is called `close_in_cs_falcon`, in ServiceNowV2 it is `close_ticket` and so on.
Maybe here a flag that is sent to the command will be a better solution.